src/components/Zoom/Zoom.js

```diff
@@ -90,6 +115,9 @@ class Zoom extends Component {
         >
           {zoomInChildren}
         </Button>
+        {zoomSlider ? (
+          <div className="wkp-zoomslider-wrapper" ref={this.ref} />
```

Comment:
we style use `tm-` prefixes

Reply:
In `trafimage-maps` use `wkp-` which stands for "Webkartenportal".
This is because sadly the `tm-` prefix is used in react-spatial. :disappointed:

Reply:
it's just a replace, if you want to change it go ahead. :)